### PR TITLE
Update documents of facets

### PR DESCRIPTION
**Pull request by satoshi-lifi**

### CODE REVIEW

File: docs/CBridgeFacet.md [hunk 1]
```diff
@@ -53,7 +41,7 @@ The swap library can be found [here](../src/Libraries/LibSwap.sol).
 
 ## LiFi Data
```
Comment (H3xept) [inline docs/CBridgeFacet.md:42]: This is present in all facets, should this be ## BridgeData? 
Also the text below says "some methods" where in fact we always pass the bridge data as param?

**Comment by ezynda3** [comment]
Can we move the `memory` to `calldata` changes to a different PR? We've already deployed all of these contracts to production and we need to figure out if it's even worth it to redeploy those at the moment.

**Review by ezynda3 — COMMENTED** [review]
Can we move the memory to calldata changes to a different PR? We've already deployed all of these contracts to production and we need to figure out if it's even worth it to redeploy those at the moment.

**Comment by satoshi-lifi** [comment]
> Can we move the memory to calldata changes to a different PR? We've already deployed all of these contracts to production and we need to figure out if it's even worth it to redeploy those at the moment.

I think even we can do it in this PR since it's only change with the `memory` to `calldata` and no logic change.
or do you want me to do it later in other PR?

**Pull request merged**